manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.locale.en-US.yaml

```diff
@@ -0,0 +1,19 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: freifunkMUC.FreifunkMeet
+PackageVersion: 2024.3.0
+PackageLocale: en-US
+Publisher: Freifunk Muenchen
+PublisherUrl: https://github.com/freifunkMUC
+PackageName: FreifunkMeet
+PackageUrl: https://github.com/freifunkMUC/jitsi-meet-electron
+License: Apache-2.0
+LicenseUrl: https://github.com/freifunkMUC/jitsi-meet-electron/raw/main/LICENSE
+Copyright: Copyright © 2024 Freifunk Muenchen
+ShortDescription: 'Jitsi Meet desktop application powered by :electron:'
+Tags:
+- hacktoberfest
```

Comment:
should be the same as: https://github.com/microsoft/winget-pkgs/blob/master/manifests/j/Jitsi/Meet/2024.3.0/Jitsi.Meet.locale.en-US.yaml#L18-L23 ?

Reply:
Maybe it should?